src/Volume.ts

```diff
@@ -176,7 +176,7 @@ export default class Volume {
     }
 
     if (shouldReload) {
-      this.loadNewData(onChannelLoaded);
+      await this.loadNewData(onChannelLoaded);
```

Comment:
I checked both `vole-core` and `vole-app`, and currently nothing is calling `await updateRequiredData` or is doing anything with the `Promise` it returns. I also consulted with @frasercl and we agreed that doing this seems okay.

Reply:
we absolutely can not harm performance during time series playback in VolE app